src/components/ConnectButton/ConnectButton.tsx

```diff
@@ -20,7 +20,9 @@ const {
 type web3StatusType = 'disconnected' | 'pending' | 'only_web3';
 
 const ConnectButton: React.FC<ConnectButtonProps> = ({
+    chainId = undefined,
```

Comment:
Do you need to assign `undefined`?

I believe it is the same thing to have
```
chainId,
moralisAuth = true,
signingMessage,
```

Reply:
it's removed (key and value) using `JSON.stringify`
if I dont use undefined i believed it'll be passed as an empty string to the backend, and i'm not sure if an empty string is being handled on the server. @BillyG83